v03_pipeline/lib/misc/validation.py

```diff
@@ -56,6 +56,30 @@ def validate_expected_contig_frequency(
             raise SeqrValidationError(msg)
 
 
+def validate_imputed_sex_ploidy(
+    mt: hl.MatrixTable,
+    sex_check_ht: hl.Table,
+) -> None:
+    mt = mt.select_cols(
+        discrepant=(
+            (
+                # All calls are diploid but the sex is Male/Unknown
+                hl.agg.all(mt.GT.is_diploid())
+                & (sex_check_ht[mt.s].predicted_sex != Sex.FEMALE.value)
+            )
```

Comment:
hmm I remember from the docs that only confirmed male XY samples have ploidy altered, and I don't think anything else has ploidy touched. I think we'd expect Unknown to be fully diploid. Do we want to throw an error if it's unknown regardless?

Reply:
hrm, I think you're correct that we should be more explicit about Unknowns.  

We have [this function](https://github.com/broadinstitute/seqr-loading-pipelines/pull/780/files#diff-bfb21e2b0d4e857967483c2ef096c5814dc2644bb9aea28496af70c52dd7fe37R14) that will throw even _earlier_ in the pipeline if there's too many Unknown in the imputed sex file... but now that we're not running a statistical method ourselves, maybe we should just throw if anything is Unknown?